### ISSUE TITLE
[Suggestion] Add setting to not randomize into animals

### ISSUE THREAD

**Issue by Chris6ix** [Parik27/V.Rainbomizer#48]
I like having the "RandomizePlayer" setting on, but sometimes that randomizes you into an animal, which is kinda stupid since it makes certain missions impossible (Because you have to climb a ladder for example) and also because the animations are really messed up and look weird.

A setting that would prevent the player from being randomized into an animal would be great.
What would be even better is a setting that just generally prevents all humans from being turned into animals and all animals from being turned into humans.

**Comment by Fryterp23** [comment]
You can do the following to prevent animals:

Add the animal models to NSFW_Models.txt in rainbomizer/data and you (or any other ped) won't be randomized into an animal.

Furthermore, you can add them to PedsBlacklist.txt in rainbomizer/data (you have to create it) if you don't want animals to be randomized into humans.

**Comment by Chris6ix** [comment]
What animal models? Where are the names?

**Comment by Parik27** [comment]
Animals can climb ladders. Also generally you can fly over them if you are a bird. We made sure you are able to shoot and enter vehicles as animals. 

Names are here: (e.g. **a_c_boar**) 
https://docs.fivem.net/docs/game-references/ped-models/

**Comment by Chris6ix** [comment]
My experience with ladders is different. I was randomized into a mountain lion in the mission "Nervous Ron" and even after trying for 2 or 3 minutes I was unable to climb the ladder to get on to the tower, so the mission was impossible to complete.
I had to fail the mission, exit it and restart it, I got randomized into a random ped, and I was able to climb the ladder without issues.

Thank you for the link to the names though. 
Just a small suggestion is to include that link as a comment with a small explanation in the config or in the NSFW_Models.txt file, so that people can easily find the model names without having to open an issue.
